init-service/src/main/java/uk/ac/ox/ndph/mts/init_service/config/WebClientConfig.java

```diff
@@ -0,0 +1,100 @@
+package uk.ac.ox.ndph.mts.init_service.config;
+
+import io.netty.channel.ChannelOption;
+import io.netty.handler.timeout.ReadTimeoutHandler;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.cloud.client.loadbalancer.LoadBalanced;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.client.reactive.ClientHttpConnector;
+import org.springframework.http.client.reactive.ReactorClientHttpConnector;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.netty.http.client.HttpClient;
+import reactor.util.retry.Retry;
+
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Supplier;
+
+@Configuration
+@ConfigurationProperties(prefix = "http-client")
+public class WebClientConfig {
```

Comment:
Any reason why this was moved to config package from configuration package ?

Reply:
configuration package is going to be deleted in the next PR.